### PR TITLE
cloud: update PoC entries 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-poc.md [hunk 1]
```diff
@@ -58,13 +58,16 @@ You might also be interested in using [TiFlash](https://docs.pingcap.com/tidb/st
...
-2. Log in to your [TiDB Cloud](http://tidbcloud.com) account. The plan selection page is displayed by default.
-3. Click **Apply for a PoC Trial** and refer to [Quick Start](/tidb-cloud/tidb-cloud-quickstart.md) to create a [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) cluster for the PoC.
+1. Fill in the PoC application form by doing one of the following:
 
-The **Proof of Concept plan** is a 14-day free trial option specifically designed for the PoC purpose. Selecting this plan takes you to the PoC application form. Once you submit the form, the TiDB Cloud support team will review your application, contact you, and transfer trial points to your account once the application is approved. You can also contact a PingCAP support engineer to assist with your PoC procedures to ensure the PoC runs as smoothly as possible.
+    - If you have already [created a Developer Tier](/tidb-cloud/tidb-cloud-quickstart.md#step-1-create-a-tidb-cluster) (one-year free trial), an prompt bar about submitting your PoC application is displayed in TiDB Cloud console. You can click the PoC application link in the bar to fill in the PoC application form.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-poc.md:65]:
```suggestion
    - If you have already [created a Developer Tier](/tidb-cloud/tidb-cloud-quickstart.md#step-1-create-a-tidb-cluster) (one-year free trial), an prompt bar about submitting your PoC application is displayed in the TiDB Cloud console. You can click the PoC application link in the bar to fill in the PoC application form.
```

File: tidb-cloud/tidb-cloud-poc.md [hunk 2]
```diff
@@ -58,13 +58,16 @@ You might also be interested in using [TiFlash](https://docs.pingcap.com/tidb/st
...
+    - If you have not created a Developer tier yet, go to the [Apply for PoC](https://en.pingcap.com/apply-for-poc/) page to fill in the PoC application form.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-poc.md:66]:
```suggestion
    - If you have not created a Developer Tier yet, go to the [Apply for PoC](https://en.pingcap.com/apply-for-poc/) page to fill in the PoC application form.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by zhoubasten — APPROVED** [review]
LGTM

**Comment by jinwhu** [comment]
LGTM

**Review by lulukelu — APPROVED** [review]

**Review by ljun0712 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by zhoubasten — APPROVED** [review]
LGTM

**Pull request merged**